code/modules/tutorials/flock_tutorial.dm

```diff
@@ -0,0 +1,449 @@
+/area/tutorial/flock
+	name = "Flock Tutorial Zone"
+	icon_state = "yellow"
```

Comment:
Checking out blob vars, there's also sound_group and dont_log_combat?

Reply:
Have put `dont_log_combat` on the parent, `sound_group` is for ambient sounds and I don't think flock has any so that can stay blank for now.